mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -151,6 +148,8 @@ def _azure_upload_file(self, credentials, local_file, artifact_path):
         Finally, since the prevailing credentials could expire in the time between the last
         stage_block and the commit, a second try-except block refreshes credentials if needed.
         """
+        from azure.core.exceptions import ClientAuthenticationError
```

Comment:
This would mean the mlflow base library may not work for databricks scenarios by default.